### PR TITLE
chore: add github issue, pr template

### PR DESCRIPTION
**Pull request by googs1025**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
- When participants are involved, it leads to more standardized issue and pull request submissions, helping reviewers better understand the meaning behind the issues and pull requests.
#### Which issue(s) this PR fixes:

Fixes # None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_request.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+- Kubernetes version (use `kubectl version`):
+- Kueue version (use `git describe --tags --dirty --always`):
```
Comment (kannon92) [inline .github/ISSUE_TEMPLATE/bug_request.md:24]:
```suggestion
- JobSet version (use `git describe --tags --dirty --always`):
```

File: .github/ISSUE_TEMPLATE/bug_request.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- Cloud provider or hardware configuration:
+- OS (e.g: `cat /etc/os-release`):
```
Comment (kannon92) [inline .github/ISSUE_TEMPLATE/bug_request.md:26]: Can we drop OS and Kernel?

Comment (googs1025) [reply .github/ISSUE_TEMPLATE/bug_request.md]: done

File: .github/ISSUE_TEMPLATE/bug_request.md (hunk 1 above)
Comment (googs1025) [reply .github/ISSUE_TEMPLATE/bug_request.md]: oh, my bad. fixed

**Comment by googs1025** [comment]
/assign @danielvegamyhre 
thanks!

**Comment by danielvegamyhre** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**